File: sukh_1684/problems/queries_on_number_of_points_inside_a_circle/solution.py
class Solution:
    def countPoints(self, points: List[List[int]], queries: List[List[int]]) -> List[int]:
        answer = []
        for qx,qy,r in queries:
            count = 0
            for px,py in points:
                if (math.sqrt((qx-px)**2 + (qy - py)**2) <= r):
                    count += 1
            answer.append(count)
        return answer
        

File: sukh_1684/problems/queries_on_number_of_points_inside_a_circle/solution.js
/**
 * @param {number[][]} points
 * @param {number[][]} queries
 * @return {number[]}
 */
var countPoints = function(points, queries) {
    let ans = [];
    queries.forEach(([qx,qy,r]) => { 
        let count = 0;
        points.forEach(([x,y]) => {
            if(Math.sqrt((qx-x)**2 + (qy-y)**2) <= r){
                    count ++;
            }
        }) ;
        ans.push(count);
    });
    return ans;
};